influxql/parser_test.go

```diff
@@ -1627,24 +1627,14 @@ func TestParser_ParseStatement(t *testing.T) {
 		{
 			s: `CREATE DATABASE testdb`,
 			stmt: &influxql.CreateDatabaseStatement{
-				Name:                  "testdb",
-				IfNotExists:           false,
-				RetentionPolicyCreate: false,
-			},
-		},
-		{
-			s: `CREATE DATABASE IF NOT EXISTS testdb`,
-			stmt: &influxql.CreateDatabaseStatement{
-				Name:                  "testdb",
-				IfNotExists:           true,
+				Name: "testdb",
```

Comment:
Then never mind! I had noticed `go fmt` doing this a few times before when I was writing code, but I never knew what triggered it.